shared/common-adapters/plain-input.native.tsx

```diff
@@ -58,11 +58,13 @@ class PlainInput extends React.PureComponent<InternalProps> {
 
     // this is a very hacky workaround for internal bugs in RN TextInput
     // write a stub with different content
-    this.setNativeProps({text: ''})
```

Comment:
This hack doesn't work when you inject an emoji on mobile currently, it'll take the blank and then ignore the actual text. RN is currently super broken in this regard and we likely should try and move to a controlled input (depending on the perf). The RN repo is littered with issues about broken selection and selection callbacks and setNativeProps calls